eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc

```diff
@@ -0,0 +1,383 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2019 Ahmet Kokulu
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/form-factors/baryonic.hh>
+#include <eos/rare-b-decays/charm-loops.hh>
+#include <eos/rare-b-decays/lambdab-to-lambdac-l-nu.hh>
+#include <eos/utils/complex.hh>
+#include <eos/utils/destringify.hh>
+#include <eos/utils/integrate-impl.hh>
+#include <eos/utils/kinematic.hh>
+#include <eos/utils/log.hh>
+#include <eos/utils/memoise.hh>
+#include <eos/utils/model.hh>
+#include <eos/utils/options.hh>
+#include <eos/utils/power_of.hh>
+#include <eos/utils/private_implementation_pattern-impl.hh>
+#include <eos/utils/integrate.hh>
+#include <eos/utils/save.hh>
+#include <eos/utils/options-impl.hh>
+
+#include <cmath>
+#include <functional>
+#include <map>
+#include <string>
+
+namespace eos
+{
+    using std::norm;
+
+    namespace lambdab_to_lambdac_l_nu
+    {
+        struct Amplitudes
+        {
+            complex<double> a_perp_0_L, a_perp_0_R;
+            complex<double> a_para_0_L, a_para_0_R;
+            complex<double> a_perp_1_L, a_perp_1_R;
+            complex<double> a_para_1_L, a_para_1_R;
+            complex<double> a_S_perp,   a_S_para;
+            complex<double> a_P_perp,   a_P_para;
+            double alpha;
+        };
+
+        struct AngularObservables
+        {
+            std::array<double, 10> _k;
+
+            AngularObservables(const Amplitudes & a)
+            {
+                // extend to include full basis
```

Comment:
Is this comment still relevant? I.e., are there further observables beyond the 10 implemented here?